### DISCUSSION TITLE
Jenkins

### DISCUSSION THREAD

**Discussion by mredodos** [coollabsio/coolify#3740] [Service Template Requests]
Can you add please https://www.jenkins.io/

https://hub.docker.com/r/jenkins/jenkins

https://www.cloudbees.com/blog/how-to-install-and-run-jenkins-with-docker-compose

**Comment by aybasaran** [comment]
this would be nice <3

**Comment by mredodos** [comment]
is on Release v4.0.0-beta.361 - v4.0.0-beta.361
